### PR TITLE
Bugfix: Go To source broken for nested classes.

### PR DESCRIPTION
**Pull request by ratcashdev**
This PR is part of the fixes needed for https://issues.apache.org/jira/browse/NETBEANS-6041
This is a fix for gradle projects, but since it's also broken for MAVEN (and possible standard netbeans projects) might need to be refactored to a different common module to fix it everywhere at once. Not sure what would be the best place.

### CODE REVIEW

**Comment by mbien** [comment]
added milestone NB14 to keep this on the radar.

**Comment by matyasberry** [comment]
NB14? ...been waiting for 4 months. Could have been included already in 12.6

**Comment by ratcashdev** [comment]
NB14? ...been waiting for 4 months. Could have been included already in 12.6

**Comment by mbien** [comment]
> NB14? ...been waiting for 4 months. Could have been included already in 12.6

probably, but nobody reviewed it yet unfortunately. NB14 is the next realistic milestone, NB13 would be still possible in theory but this would have to get reviewed quickly and rebased on top of the delivery branch since NB13 is in the rc phase.

If you really want to get a PR in and see no activity here, dropping a mail to the dev list can sometimes help to get some eyes on it. 

Thanks for sticking around.

**Comment by ratcashdev** [comment]
will do next time. Tried to make it clear in the jira ticket that there's a PR, but that, apparently, was left unattended too.

**Comment by lkishalmi** [comment]
I'm sorry, I've totally missed this one.

File: java/gradle.test/src/org/netbeans/modules/gradle/test/ui/nodes/GradleJUnitNodeOpener.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public void openTestsuite(TestsuiteNode tn) {
             Node first = children.getNodeAt(0);
             if ((first != null) && (first instanceof GradleTestMethodNode)) {
                 GradleTestMethodNode node = (GradleTestMethodNode) first;
-                Location loc = new Location(node.getTestLocation().getFileName());
+								Location loc = node.getTestLocation().withNoTarget();
```
[the remark this answers is not in the document]
Comment (matyasberry) [reply java/gradle.test/src/org/netbeans/modules/gradle/test/ui/nodes/GradleJUnitNodeOpener.java]: Well, yes. With today's merge tools and their ability to hide whitespace, formatting became much less of a concern for me.
Thanks for putting it into shape and merging. So this is... for 14, correct?

[the remark this answers is not in the document]
Comment (ratcashdev) [reply java/gradle.test/src/org/netbeans/modules/gradle/test/ui/nodes/GradleJUnitNodeOpener.java]: Well, yes. With today's merge tools and their ability to hide whitespace, formatting became much less of a concern for me.
Thanks for putting it into shape and merging. So this is... for 14, isn't it?

**Comment by ratcashdev** [comment]
@lkishalmi do you think this  can be built out of the current master and dropped into 13.0 as a jar?

**Comment by lkishalmi** [comment]
Not really. Though you can try to build nbm-s out of the gradle,java and gradle.junit modules, though you may need to build the gradle main module as well.

**Pull request merged**